src/command/administration.py

```diff
@@ -197,9 +196,10 @@ async def cmd_user_info_or_callback_set_user(event: Union[events.NewMessage.Even
                        data=f"set_user={user_id},0") if user.state != 0 else inner.utils.emptyButton,),
         (Button.inline(f"{i18n[lang]['set_user_state_as']} \"{i18n[lang]['user_state_1']}\"",
                        data=f"set_user={user_id},1") if user.state != 1 else inner.utils.emptyButton,),
-        (Button.inline(f"{i18n[lang]['reset_sub_limit_to_default']} "
+        (inner.utils.emptyButton if is_default_limit else
+         Button.inline(f"{i18n[lang]['reset_sub_limit_to_default']} "
                        f"({default_sub_limit if default_sub_limit > 0 else i18n[lang]['sub_limit_unlimited']})",
-                       data=f"reset_sub_limit={user_id}") if not is_default_limit else inner.utils.emptyButton,),
+                       data=f"reset_sub_limit={user_id}"),),
```

Comment:
To keep the same pattern as the previous lines, the condition should not be reversed.